client/page_main.ui

```diff
@@ -10,6 +10,12 @@
     <height>397</height>
    </rect>
   </property>
+  <property name="minimumSize">
+   <size>
+    <width>821</width>
+    <height>397</height>
```

Comment:
These numbers look quite arbitrary, is there a particular reason for this choice?

Reply:
I just took the same values as the base form geometry